src/netappfiles-preview/azext_netappfiles_preview/_help.py

```diff
@@ -13,454 +13,3 @@
     type: group
     short-summary: Manage Azure NetApp Files (ANF) Resources.
 """
```

Comment:
You can add the following two line to overwrite the help definition in azure-cli.

```suggestion
"""

helps['netappfiles volume create'] = None

helps['netappfiles volume update'] = None

```

Reply:
Thanks, will look into that.

This does however seem as a shortcoming of the linter tho, since this extension overwrites the cli.  Also it is flagging paramters like `--file-path` for example as invalid while it is valid in both the main Azure CLI and this extension.

<img width="341" alt="image" src="https://github.com/Azure/azure-cli-extensions/assets/3875306/373d2984-9e17-4720-8ce0-da5e23452f99">